### PR TITLE
Add travis, refactor, and tests

### PR DESCRIPTION
**Pull request by litzenberger**

### CODE REVIEW

**Comment by litzenberger** [comment]
Coverage: 91.15% (10/113)

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+language: node_js
+node_js:
+- '6'
+- '7'
```
Comment (mcollina) [inline .travis.yml:4]: we need node 8 here.

File: pino-eventhub.js [hunk 2]
```diff
@@ -57,10 +57,9 @@ function pinoEventHub (opts) {
     return function inner(response) {
       debug('response.statusCode =', response.statusCode)
       debug('response.statusMessage =', response.statusMessage)
-
       if (response.statusCode != 201) {
-        // splitter.emit(`response error =`, response.statusMessage)
-        console.log(`response error =`, response.statusMessage)
+        splitter.emit('error', new Error(response.statusCode ))
+        console.log('Error',response.statusMessage)
```
Comment (mcollina) [inline pino-eventhub.js]: Do we need this `console.log` statement?

**Comment by mcollina** [comment]
Can you lint it with standard?

**Review by mcollina — APPROVED** [review]
LGTM with the nits fixed

File: pino-eventhub.js (hunk 2 above)
Comment (litzenberger) [reply pino-eventhub.js]: no, removing it.

File: .travis.yml (hunk 1 above)
Comment (litzenberger) [reply .travis.yml:4]: node 8 added.

**Comment by litzenberger** [comment]
I addressed node 8, lint, and the console.log notes.  Added coveralls also but not sure if that's working.

**Pull request merged**